google-cloud-storage/src/main/java/com/google/cloud/storage/testing/RemoteStorageHelper.java

```diff
@@ -67,6 +70,33 @@ public StorageOptions getOptions() {
     return options;
   }
 
+  public static void cleanOldBuckets(final Storage storage, final long olderThan, long timeoutMs) {
```

Comment:
"old" is relative here, and is effectively determined by the `olderThan` parameter. I suggest just `cleanBuckets`.